spynnaker_integration_tests/test_spike_source/test_variable_rate.py

```diff
@@ -272,6 +277,6 @@ def test_variable_rate_100us(self):
 
 
 if __name__ == '__main__':
-    # variable_rate_options()
+    variable_rate_options()
     # variable_rate_reset()
-    variable_rate_100us()
+    # variable_rate_100us()
```

Comment:
Any reason why this (0.1 timestep) is no longer tested?

Reply:
Oh I must have done that during debugging.  Note that this part of the code doesn't stop the test happening, just when run without pytest...